### ISSUE TITLE
Usage with React 15?

### ISSUE THREAD

**Issue by ibrahima** [compulim/react-sanitized-html#7]
Hi, I noticed that the peer dependencies now specify React 16, but it doesn't really look like it should require React 16 for anything yet. Does this also work on React 15?

Thanks!

Edit: Looks like it works fine on React 15.6, so perhaps the peer dependencies can be loosened? I think the main thing is that you need a version that has PropTypes in its own package, which has been the case since 15.5 I believe.

**Comment by compulim** [comment]
Punting this bug as React@16 was release more than a year ago.

**Comment by ibrahima** [comment]
Ah, I understand. Well, we haven't been able to upgrade to React 16, not sure about other shops, so it seems like it'd be nice to reduce the restriction but it doesn't matter for me anyway because I didn't end up using this package (for now).

**Comment by compulim** [comment]
Thanks @ibrahima. Let me see if I can get React 15 in the next release. I was working in a shop so I know how much pain it was to upgrade a core component.

Even you might not be using the component by now, feel free to comment on it (API or usage) and navigate around my other components. 😉

I was recently working on some speech components and they may be good for your app/tool/website.

**Comment by ibrahima** [comment]
Sure, thanks! No rush on my end, but I think when I tried it I was able to just install it anyway and it seemed to work fine on 15.6. The issue with big upgrades like React 16 is that all your dependencies need to support it, so in some cases we might be stuck waiting for some dependencies to update (I am not sure if this is still true, but it was earlier and we haven't had time to look into it lately).
